doc/index.rst

```diff
@@ -33,17 +33,64 @@ Documentation organisation
 Citing Arbor
 ------------
 
-The Arbor software can be cited via Zenodo: |zlatest|
+The Arbor software can be cited by version via Zenodo or via Arbors introductory paper.
+
+Latest version
+    |zlatest|
+
+Version 0.5
+    |z05|
+
+    .. code-block:: latex
+
+        @software{nora_abi_akar_2021_4428108,
+        author       = {Nora Abi Akar and
```

Comment:
Do these render nicely in bibtex? I never remember where to put "" and {}.

Reply:
Good question. It's a copy paste from Zenodo, I added the extra brackets for Nora's last name and verified that it renders well.